components/patient-data/impl/src/main/resources/ApplicationResources.properties

```diff
@@ -14,6 +14,8 @@ PhenoTips.PatientClass_omim_id_hint=Generally, a disorder is used to annotate th
 PhenoTips.PatientClass_select_phenotypes_hint=For each phenotype, choose the most specific term possible, based on the definition (available by clicking the \u201Ci\u201D) and not necessarily simply based on the label. The more specific the item chosen, the better the specificity of the whole phenotypic profile. Record \u201CY\u201D for Yes, to indicate a phenotype being present in the patient. You may provide annotations for things that were investigated and specifically not observed by clicking on the \u201CN\u201D for No. Choose terms that are thought to be pathological or unusual, even if they are not so in the greater populace. For example, \u2018blue irides\u2019 is a common phenotype in the population, but as a phenotype annotation, it is a subtype of \u2018abnormal iris pigmentation\u2019 and should only be chosen in cases where abnormal pigmentation is suspected, such as \u2018Heterochromia\u2019.
 PhenoTips.InvestigationClass_hint=List genes considered as candidates for the patient's disease.
 PhenoTips.RejectedGenesClass_hint=List genes that have been investigated and ruled out as candidates for the patient's disease.
+PhenoTips.PatientClass_solvedCase_hint=Check this box if the molecular diagnosis for this patient is known. Pubmed ID may be entered for any published results as well as the solved gene and resolution notes.
```

Comment:
"Check this box is the molecular diagnosis for the patient is known."
Leave the rest off. We can add a hint for those fields later, if we'd like, but the solved gene field will be gone soon anyway.